module/multimedia/src/Infrastructure/Provider/MultimediaExtensionProvider.php

```diff
@@ -43,4 +42,34 @@ public function dictionary(): array
             'pptx',
         ];
     }
+
+    /**
+     * @return string[]
+     */
+    public function mimeDictionary(): array
+    {
+        return [
```

Comment:
Wouldn't creating a relationship dictionary between extension and mime type be better ??